scripts/updates/auto-updates.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+set -e -u
+BASEDIR=$(dirname "$(realpath "$0")")
+PKGSDIR="${BASEDIR}/../../packages"
+
+# These variables should be set in environment outside of this script.
+# Build updated packages.
+: "${BUILD_PACKAGES:=false}"
+# Commit changes to Git.
+: "${GIT_COMMIT_PACKAGES:=false}"
+# Push changes to remote.
+: "${GIT_PUSH_PACKAGES:=false}"
+
+if [ -z "${GITHUB_API_TOKEN-}" ]; then
+	echo "You need a Github Personal Access Token be set in variable GITHUB_API_TOKEN."
+	exit 1
+fi
+
+for package in $(ls -1 "${PKGSDIR}"); do
+	unset auto_update_flag project version_regexp termux_version termux_epoch latest_version srcurl
+
+	# Extract some build vars
+	build_vars=$(set +e +u; . "${PKGSDIR}/${package}/build.sh" 2>/dev/null; \
+		echo "auto_update_flag=${TERMUX_PKG_AUTO_UPDATE}"; \
+		echo "termux_version=${TERMUX_PKG_VERSION}"; \
+		echo "srcurl=${TERMUX_PKG_SRCURL}" ;\
+	)
+
+	# Eval build vars to local variables
+	source <(echo "$build_vars")
+
+	if [ "${auto_update_flag}" != "true" ]; then
+		echo "[SKIP] ${package} is not auto updatable."
```

Comment:
If package doesn't support auto updates, omit it. Don't print any useless messages. Remember that CI logs are subject for reviewing and any extra useless information makes it harder.